### ISSUE TITLE
Not able to do "mvn clean install" due to Lombok error on Windows setup

### ISSUE THREAD

**Issue by wlin98004** [stamp-web/stamp-imageparsing#76]
Got clean source from GIT and not able to do "mvn clean install" on Windows.  Works fine with "Linux".  

[ERROR] Failed to execute goal [32morg.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile[m [1m(default-compile)[m on project [36mstamp-imageparsing[m: [1;31mFatal error compiling[m: java.lang.IllegalAccessError: class lombok.javac.apt.LombokProcessor (in unnamed module @0x4975dda1) cannot access class com.sun.tools.javac.processing.JavacProcessingEnvironment (in module jdk.compiler) because module jdk.compiler does not export com.sun.tools.javac.processing to unnamed module @0x4975dda1 -> [1m[Help 1]

Can be resolved by specifying a specific version for Lombok.

```
		<dependency>
			<groupId>org.projectlombok</groupId>
			<artifactId>lombok</artifactId>
			<optional>true</optional>
			<version>1.18.22</version>
			<scope>provided</scope>
		</dependency>
```

**Comment by jadrake75** [comment]
I suspect you are not using the JDK but using a JRE - if you are in eclipse or other IDE did you make sure to point to a JDK ?

**Comment by wlin98004** [comment]
I am using JDK.  Here is my JAVA_HOME setting.  I am still getting the following error.

`JAVA_HOME=C:\Program Files\Java\jdk-18.0.2.1`

![image](https://user-images.githubusercontent.com/16296857/194188507-d3839bde-6cad-4d20-8be4-9639bd911534.png)

If I fixed the Lombok version to 1.18.24, then it fixes the issue.  I don't think it is a good idea to applied a fixed version.

**Comment by jadrake75** [comment]
So I looked at the maven pom.xml.  So lombok is marked as provided.  This means it is not considered a "direct" dependency but is provided to the environment (think of it like requirements).  So we properly should just update the README to state the version provided.  In all honestly I usually build this from the STS eclipse where it is provided.  I noticed on my Linux builds I am using 1.18.8 but I have no issue with us adding a version to the pom.xml

Can you put up a small PR to add this to the pom.xml for this dependency
`<version>[1.18.24,)</version>`

This syntax (took me a while to find https://www.baeldung.com/maven-dependency-latest-version indicates 1.18.24 is the minimum version needed
